### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Thewm**

### CODE REVIEW

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+          protein: 0,
+          iron: "2%"
+        }
+      ]
+    };
+  },
+  computed: {
+    paginate() {
+      let pages = Math.ceil(this.items.length / 10);
+      if (pages <= 1) {
```
Comment (pmzi) [inline src/views/Dashboard.vue:233]:
```
return pages >1;
```

Comment (pmzi) [reply src/views/Dashboard.vue:233]: I can't figure out what it is doing by reading its name, which is "paginate"! What do you mean by that?

File: src/views/Dashboard.vue [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        return false;
+      } else {
+        return true;
+      }
+    },
+    pageCount() {
+      let pageCounter = Math.ceil(this.items.length / 10);
```
Comment (pmzi) [inline src/views/Dashboard.vue:240]: Is this variable really necessary?

File: src/views/Dashboard.vue [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+      return pageCounter;
+    }
+  },
+  methods: {
+    handlePage(value) {
+      this.page = value;
+      if (this.page === 1) {
+        this.$router.push("/articles");
```
Comment (pmzi) [inline src/views/Dashboard.vue:248]: It is highly recommended to use an object for routing. For example:
```
this.$router.push({.name: 'sth' });
```

File: src/views/NotFound.vue [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        <v-col>
+            <h3 class="display-2"> <router-link :to="{ name: 'Login' }" class="green--text text-decoration-none">Back to safety <v-icon class="green--text display-2">mdi-arrow-collapse-right</v-icon> </router-link> </h3>
+        </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+export default {
+    name: "not-found"
```
Comment (pmzi) [inline src/views/NotFound.vue:16]: The names of the component should be exactly the same as their file system name. In this case, it should be NotFound.

File: src/views/Register.vue [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+            <span class="ml-5 mb-n2">User</span>
+            <v-col cols="11">
+              <v-text-field
+                class="rounded"
+                dense
+                flat
+                outlined
+                solo
+                autocomplete
+                v-model="user_name"
```
Comment (pmzi) [inline src/views/Register.vue:31]: any JS variable should be camelCase. I will extrapolate further at the end.

**Review by pmzi — CHANGES_REQUESTED** [review]
My very preliminary review has been finished. We have at least 2 more reviews. First of all, take a look at the provided comments, then, we will discuss those comments I said I'm going to explain more.

The next phase of the review will be from the architectural view. Which is way more important than overall review.

The most important thing is to config eslint and prettier for this phase. It will be much helpful.

File: src/router/index.js [hunk 6]
```diff
@@ -1,6 +1,8 @@
 import Vue from "vue";
```
[the remark this answers is not in the document]
Comment (Thewm) [reply src/router/index.js:1]: I can't figure out how can i do that? I googled it and got some weird results

**Pull request merged**